### ISSUE TITLE
ProgressEvent on android not work 

### ISSUE THREAD

**Issue by hejiacong** [Gamua/Starling-Framework#722]
I think this is not problem about Starling,i post this to Starling forum but without response,so i post it here to get more attention
```as3
var loader:Loader=new Loader();
loader.addEventListener(ProgressEvent.PROGRESS, onProgress);
loader.load(new URLRequest("a local file url"));
var tf:TextField=new TextField();
stage.addChild(tf);
 
function onProgress(event:ProgressEvent):void
{
    trace(event.bytesLoaded/event.bytesTotal);
    tf.text=String(event.bytesLoaded/event.bytesTotal);
}
```
the problem is
the trace is print info correctly,but the textfield is not update until progress finish,i think the code is alway running in the onProgress function in one frame,so there is no time for updating the ui,may be load local resource is too fast to update ui.
Such problem is only happen on android, ios is ok.
I have report this problem to Adobe two week ago,but without any response right now.
https://bugbase.adobe.com/index.cfm?event=bug&id=3990146

Because AssetManager use URLLoader to load resource,so when i load large resource,the application is block until load finish and can't show any progress info to user,it is very bad experience

i can use another way to solve such problem for local file,it work well!but AssetManager not use FileStream,and remote resource still a problem.i don't known how to fit this problem right now.
```javascript
var fileStream:FileStream=new FileStream();
fileStream.addEventListener(ProgressEvent.PROGRESS, onProgress);
fileStream.openAsync(file, FileMode.READ);
everything work well!but AssetManager not use FileStream
```

**Comment by PrimaryFeather** [comment]
Sorry for the late reply, I was on a vacation for the last two weeks, so I couldn't answer earlier.

I'm not sure I understand the problem correctly. You say that the trace output does appear correctly, right? Because the Textfield can be updated only once per frame, as you suspected. So that behavior would be fine (read: as expected).

But if you'd get a "onProgress" call once per frame when loading assets with the AssetManager, that should be sufficient, shouldn't it?

Perhaps you can rephrase the problem so that I understand it better. Thanks in advance!

**Comment by hejiacong** [comment]
Thank you for your quick response!
 a runnable app source code is here
```javascript
package {
import flash.display.Sprite;
import flash.events.ProgressEvent;
import flash.events.TimerEvent;
import flash.net.URLLoader;
import flash.net.URLLoaderDataFormat;
import flash.net.URLRequest;
import flash.text.TextField;
import flash.utils.Timer;

public class test extends Sprite {

public function test()
{
trace("hello");
var tf:TextField=new TextField();
tf.text="0";
stage.addChild(tf);

var timer:Timer=new Timer(5000,1);
timer.addEventListener(TimerEvent.TIMER_COMPLETE, startLoad);
timer.start();

function startLoad():void{
var loader:URLLoader=new URLLoader();
loader.dataFormat=URLLoaderDataFormat.BINARY;
loader.addEventListener(ProgressEvent.PROGRESS, onProgress);
loader.load(new URLRequest("bigfile.mp4"));//Replace a big local file here
}

function onProgress(event:ProgressEvent):void
{
trace(event.bytesLoaded/event.bytesTotal);
tf.text=String(event.bytesLoaded/event.bytesTotal);
}

}

}
}
```
Run in any android device,you will see the problem.
the tf is only show 0 or 1,not the same with trace value.

I upload the scout screenshot, the progresss has been fired again and again in one frame,so the whole app has been blocked until the complete event fire,the fps drop to 0.9
I am sorry my scout view is in Chinese
![scout](https://cloud.githubusercontent.com/assets/7249583/8306706/b68c28fc-19ec-11e5-917c-e4ca138987e8.png)
![scout2](https://cloud.githubusercontent.com/assets/7249583/8306877/ece6562e-19ed-11e5-81a2-2cec752179ef.png)

**Comment by PrimaryFeather** [comment]
I will try that out as soon as I've got some time, but don't expect too much of me: as your sample proves, it's definitely an AIR problem. 

If I can reproduce the problem, I will vote for the issue, of course! But other than that, there's unfortunately not much I can do. :pensive:

**Comment by hejiacong** [comment]
I see，i just want to get more attention about this bug!
Adobe reponse always so slow,so disappointing!
Maybe i will create an new issue to adobe bugbase,and describe it more and hope to get more attention,
I think it is very serious bug,once load something,the app always block,very very bad experience!
Thanks again Daniel!

**Comment by PrimaryFeather** [comment]
I can totally understand what you mean — yes, Adobe's bugbase is unfortunately crowded with issues and hard to search through, so it's hard to get their attention. The best option you have is to collect as many votes as possible — perhaps you link to the bug report in your Starling forum post, as well. That get indexed by google (better than the bugbase), so when somebody else runs into this issue, he'll likely find it and can vote for it.

And, as soon as I can reproduce it, I'll do the same, of course! :-)

**Comment by PrimaryFeather** [comment]
I could reproduce this on Android — thus, I added my vote to the bugbase issue.
I'll close the issue here, if you're fine with that, though. (Since it's not a Starling issue.)

I'm crossing my fingers for you that this gets fixed!
